src/Logins/registration/Register.js

```diff
@@ -0,0 +1,93 @@
+import { useState } from "react";
+import "./register.css";
+import FormInput from "./FormInputs";
+
+const Register = () => {
+  const [values, setValues] = useState({
+    username: "",
+    email: "",
+    birthday: "",
+    password: "",
+    confirmPassword: "",
+  });
+
+  const inputs = [
```

Comment:
If this is a static value. We should add them to a constants file in utils folder.